tests/unit/nupic/encoders/delta_test.py

```diff
@@ -41,59 +43,67 @@ def setUp(self):
 
 
   def testDeltaEncoder(self):
-      """simple delta reconstruction test"""
-      for i in range(5):
-        encarr =  self._dencoder.encodeIntoArray(i, np.zeros(100), learn=True)
-      self._dencoder.setStateLock(True)
-      for i in range(5, 7):
-        encarr =  self._dencoder.encodeIntoArray(i, np.zeros(100), learn=True)
-      res = self._dencoder.topDownCompute(encarr)
-      self.assertEqual(res[0].value, 6)
-      self.assertEqual(self._dencoder.topDownCompute(encarr)[0].value, res[0].value)
-      self.assertEqual(self._dencoder.topDownCompute(encarr)[0].scalar, res[0].scalar)
-      self.assertTrue((self._dencoder.topDownCompute(encarr)[0].encoding == res[0].encoding).all())
+    """simple delta reconstruction test"""
```

Comment:
This is weird. All lines seems equal but GH insists that there were changes. Could you inform what kind of changes happened here?